.github/workflows/auto_tag.yaml

```diff
@@ -0,0 +1,33 @@
+name: Essential files modified
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - quantum/**/*
+    - tmk_core/**/*
+    - drivers/**/*
+    - tests/**/*
+    - util/**/*
+    - platforms/**/*
+    - Makefile
+    - '*.mk'
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+
+    # protect against those who develop with their fork on master
+    if: github.repository == 'qmk/qmk_firmware'
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.26.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DEFAULT_BUMP: 'none'
```

Comment:
Should this be `minor` or omitted? Seems like with this change no tagging will happen at all unless we manually `#minor` everything.

Reply:
Was trying to be smart and disable users being able to bump, but i had read the docs wrong
```suggestion
```